farm/data_handler/processor.py

```diff
@@ -389,7 +389,12 @@ def __init__(
         :param tokenizer: Used to split a sentence (str) into tokens.
         :param max_seq_len: Samples are truncated after this many tokens.
         :type max_seq_len: int
-        :param data_dir: The directory in which the train and dev files can be found. Squad has a private test file
+        :param data_dir: The directory in which the train and dev files can be found.
+                         If not available the dataset will be loaded automaticaly
+                         if the last directory has the same name as a predefined dataset.
+                         These predefined datasets are defined as the keys in the dict at
+                         `farm.data_handler.utils.DOWNSTREAM_TASK_MAP <https://github.com/deepset-ai/FARM/blob/master/farm/data_handler/utils.py>`_.
+                         Squad has a private test file
```

Comment:
Lets not talk about SQuaD or other individual datasets in here. 
GermEval18 only has train and test, no dev. I think information like this could be added directly in the example scripts (but doesnt have to be).

Reply:
- removed SQuaD from docstring TCP
- added GermEval18 dataset description to examples and `data_handler.utils.py`